submissions/yhorodechna/html-css-popup/index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>popup</title>
+    <link rel="stylesheet" href="css/null_style.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width , initial-scale=1.0" />
+</head>
+
+<body>
+    <div class="wrapper">
+        <header class="header">
+            <div class="popup__body">
+                <input tabindex="1" class="popup__input" type="checkbox" id="popup__input" />
+                <label for="popup__input" class="popup__label  menu__item">
+                    <img src="img/header/popup-button.png" alt="popup picture" />
+                </label>
+                <nav class="popup">
+                    <ul class="popup__list">
+                        <li class="popup__item" tabindex="2">
+                            <img src="img/calendar.png" alt="calendar picture" />
+                            <span>Calendar</span>
+                        </li>
+                        <li class="popup__item" tabindex="2">
+                            <img src="img/compass.png" alt="compass picture" />
+                            <span>Compass</span>
+                        </li>
+                        <li class="popup__item" tabindex="2">
+                            <img src="img/design.png" alt="design picture" />
+                            <span>Design</span>
+                        </li>
+                        <li class="popup__item" tabindex="2">
+                            <img src="img/download.png" alt="download picture" />
+                            <span>Download</span>
+                        </li>
+                        <li class="popup__item" tabindex="2">
+                            <img src="img/favorite.png" alt="favorite picture" />
+                            <span>Favorite</span>
+                        </li>
+                        <li class="popup__item" tabindex="2">
+                            <img src="img/location.png" alt="location picture" />
+                            <span>Location</span>
+                        </li>
+                        <li class="popup__item" tabindex="2">
+                            <img src="img/mail.png" alt="mail picture" />
+                            <span>Mail</span>
+                        </li>
+                        <li class="popup__item" tabindex="2">
+                            <img src="img/message.png" alt="message picture" />
+                            <span>Message</span>
+                        </li>
+                        <li class="popup__item" tabindex="2">
+                            <img src="img/phone.png" alt="phone picture" />
+                            <span>Phone</span>
+                        </li>
+                        <li class="popup__more more">
+                            <input tabindex="2" class="more__input" type="checkbox" id="more__input" />
```

Comment:
By default, all interractive elements already have a focus state, adding `tabindex` attribute you change the default order of interactive elements set by HTML markup. In the current task no need to use `tabindex` attribute unless you want to change default order)

Reply:
Deleted all tabindex.